app/models/endpoint.rb

```diff
@@ -24,4 +50,18 @@ def resolve_verify_ssl_value(val)
     else            val
     end
   end
+
+  # Returns a list, to support concatenated PEM certs.
+  def parse_certificate_authority
+    return [] if certificate_authority.blank?
+    certificate_authority.split(/(?=-----BEGIN)/).reject(&:blank?).collect do |pem_fragment|
```

Comment:
Do you have a test for concatenated PEM certs?  It seems the test only have 1 cert